src/webnn_native/ops/Conv2d.cpp

```diff
@@ -35,6 +35,32 @@ namespace webnn_native { namespace op {
         return &mOptions;
     }
 
+    void Conv2d::calculateOutputSize(int32_t inputHeight,
```

Comment:
Can we move it to utils?

Reply:
I see that all calculateOutputSize functions are written under the cpp file of the corresponding operator, so I think this can be reserved for now.